### DISCUSSION TITLE
gInk is used on OBS (Especially for tutorial recordings)

### DISCUSSION THREAD

**Discussion by brotons** [geovens/gInk#146] [Ideas]
![43 40 - 03-12-2021](https://user-images.githubusercontent.com/17508310/144536089-ded57db3-4d63-49c4-8bdf-3a01fdb1c75d.png)

Currently (9:39 AM 12/3/2021) gInk cannot be captured for recording on OBS, when selecting (Window Capture >) Source in OBS for gInk was not found.

Any solution? Immediately if it can be made because this is really needed at this time, because many people make video content.

**Comment by pubpub-zz** [comment]
you may have a look into those :
https://github.com/pubpub-zz/ppInk/issues/21
https://github.com/pubpub-zz/ppInk/discussions/128
